test/metrics/prom-metrics-linter/custom_linter_rules.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus/testutil/promlint"
+	dto "github.com/prometheus/client_model/go"
+)
+
+func CustomLinterRules(problems []promlint.Problem, mf *dto.MetricFamily, operatorName, subOperatorName string) []promlint.Problem {
+	// Check metric prefix
+	nameParts := strings.Split(*mf.Name, "_")
+
+	if nameParts[0] != operatorName {
+		problems = append(problems, promlint.Problem{
+			Metric: *mf.Name,
+			Text:   fmt.Sprintf("name need to start with '%s_'", operatorName),
```

Comment:
==> "name **must** ... "?

Reply:
I saved the promlint error msg's format.